tool_commands/commands/Step 1 - Check data/schematisation_checker.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import os
+
+from sqlalchemy.exc import OperationalError
+from threedi_modelchecker import errors
+from threedi_modelchecker.exporters import format_check_results
+from threedi_modelchecker.model_checks import ThreediModelChecker
+
+from ThreeDiToolbox.tool_commands.base.custom_command import CustomCommandBase
+from ThreeDiToolbox.tool_commands.schematisation_checker.controller import \
+    SchemaCheckerDialogWidget
+from ThreeDiToolbox.utils.user_messages import pop_up_info
+from ThreeDiToolbox.utils.user_messages import progress_bar
+
+logger = logging.getLogger(__name__)
+
+
+class CustomCommand(CustomCommandBase):
+
+    def __init__(self, iface, ts_datasource, plugin_dir):
```

Comment:
- plugin_dir is misschien niet nodig.
- ts_datasource wordt niet gebruikt, dus die hoeft ook niet meegegeven te worden (dat helpt mij weer met de ts_datasources refactor).

Reply:
`ts_datasource` wordt niet gebruikt, maar door het dynamisch importeren en runnen van `CustomCommand` (zie comment hierboven), kunnen we niet niet zomaar de `ts_datasource` weghalen uit de constructor. Want deze wordt wel gebruikt bij andere `CustomCommands`. 

Voor nu is het denk ik te veel werk om dit te refactoren en levert het te weinig op. Ik zal er voor nu een comment bij zetten om dit duidelijk te maken.